tchannel/tornado/util.py

```diff
@@ -90,3 +90,12 @@ def go():
     go()
 
     return all_done_future
+
+
+def num_out_pendings(conns):
```

Comment:
+1

Based on usage it looks like this should be a method on Peer (which acts as a grouping of connections of sorts).